translate/storage/test_pypo.py

```diff
@@ -209,11 +209,32 @@ def test_combine_msgidcomments(self):
         regenposource = str(pofile)
         assert regenposource.count("_:") == 1
 
-    def test_merge_duplicates_msgctxt(self):
-        """checks that merging duplicates works for msgctxt"""
+    def test_duplicates_default(self):
+        """checks that duplicates are ignored by default"""
         posource = '#: source1\nmsgid "test me"\nmsgstr ""\n\n#: source2\nmsgid "test me"\nmsgstr ""\n'
         pofile = self.poparse(posource)
+        assert len(pofile.units) == 1
+        assert str(pofile.units[0]).count("source1") == 1
+        assert str(pofile.units[0]).count("source2") == 0
+
+    def test_duplicates_allow(self):
+        """
+        checks that you can create a pofile with duplicates - this is useful
+        or converting to other formats
+        """
+        posource = ('#: source1\nmsgid "test me"\nmsgstr ""\n\n'
+                    '#: source2\nmsgid "test me"\nmsgstr ""\n')
```

Comment:
Please use the following instead which is much more easy to read:

```python
        posource = """#: source1
msgid "test me"
msgstr ""


#: source2
msgid "test me"
msgstr ""
"""
```

Please follow the same pattern in the newly added `posource`.